nbd-server.c

```diff
@@ -779,6 +790,7 @@ GArray* parse_cfile(gchar* f, struct generic_conf *const genconf, bool expect_ge
 		{ "multifile",	FALSE,	PARAM_BOOL,	&(s.flags),		F_MULTIFILE },
 		{ "treefiles",	FALSE,	PARAM_BOOL,	&(s.flags),		F_TREEFILES },
 		{ "copyonwrite", FALSE,	PARAM_BOOL,	&(s.flags),		F_COPYONWRITE },
+		{ "waitfile",   FALSE,	PARAM_BOOL,	&(s.flags),		F_WAIT },
```

Comment:
It's probably a good idea to make nbd-server error out when `copyonwrite` and `waitfile` are both enabled at the same time.